CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [TBD] - TBD
```

Comment:
version number?

Reply:
Yup, 16.3.3 I guess?

Reply:
can go in next release, this is a "safety" fix we're not using different database in Circus Train though.

Reply:
OK, fine, just leave it then.